x-pack/plugins/security_solution/public/common/components/event_details/columns.tsx

```diff
@@ -192,40 +186,56 @@ export const getColumns = ({
     name: i18n.VALUE,
     sortable: true,
     truncateText: false,
-    render: (values: string[] | null | undefined, data: EventFieldsData) => (
-      <FullWidthFlexGroup
-        direction="column"
-        alignItems="flexStart"
-        component="span"
-        gutterSize="none"
-      >
-        {values != null &&
-          values.map((value, i) => (
-            <FullWidthFlexItem
-              grow={false}
-              component="span"
-              key={`event-details-value-flex-item-${contextId}-${eventId}-${data.field}-${i}-${value}`}
-            >
-              <div data-colindex={3} onFocus={onFocusReFocusDraggable} role="button" tabIndex={0}>
-                {data.field === MESSAGE_FIELD_NAME ? (
-                  <OverflowField value={value} />
-                ) : (
-                  <FormattedFieldValue
-                    contextId={`event-details-value-formatted-field-value-${contextId}-${eventId}-${data.field}-${i}-${value}`}
-                    eventId={eventId}
-                    fieldFormat={data.format}
-                    fieldName={data.field}
-                    fieldType={data.type}
-                    isObjectArray={data.isObjectArray}
-                    value={value}
-                    linkValue={getLinkValue(data.field)}
-                  />
-                )}
-              </div>
-            </FullWidthFlexItem>
-          ))}
-      </FullWidthFlexGroup>
-    ),
+    render: (values: string[] | null | undefined, data: EventFieldsData) => {
+      const fieldFromBrowserField = getFieldFromBrowserField(
+        [data.category, 'fields', data.field],
+        browserFields
+      );
+      return (
+        <FullWidthFlexGroup
+          direction="column"
+          alignItems="flexStart"
+          component="span"
+          gutterSize="none"
+        >
+          {values != null &&
+            values.map((value, i) => {
+              if (fieldFromBrowserField == null) {
```

Comment:
Consider adding a similar check when rendering the field in the table view, because per the screenshot below, `process.created` will be rendered as a draggable / filterable item in the table, but Elasticsearch won't, for example return results for this field if it's added as a filter to the KQL bar:

<img width="1871" alt="process_created_in_table_view" src="https://user-images.githubusercontent.com/4459398/116943443-d8c30800-ac30-11eb-97b9-45c443df3012.png">

Reply:
I completely agree with you but per our discussion over zoom, it will be a large change and since we do not have that much time for BC3. I will offer you that the team do a follow up PR. Anyway, with this PR, we will bring back some value to our user. Can you please transform this comment as an issue with the `triage-needed` as a label for the `threat hunting team`?

Reply:
https://github.com/elastic/kibana/issues/99150